### PR TITLE
[NCP-3463] some text added

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

File: en_US.json [hunk 1]
```diff
@@ -1848,11 +1848,14 @@
 	"report.range.title.button.query": "Query",
 	"report.range.title.requests.time.range": "Requests Within Time Range",
 	"report.range.title.traffic.time.range": "Traffic Within Time Range",
+	"report.range.txt.duration": "Duraton",
```
Comment (burtonquantil) [inline en_US.json:1851]: This should be "Duration"

**Review by burtonquantil — CHANGES_REQUESTED** [review]
One word is misspelled.

File: en_US.json (hunk 1 above)
Comment (jungho73) [reply en_US.json]: I fixed the word misspelled.
Thanks.

**Review by chiraags — APPROVED** [review]
Good to merge.

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Pull request merged**